ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Deployment/DeploymentPage.tsx

```diff
@@ -42,17 +43,18 @@ const deploymentMetadataQuery = gql`
 `;
 
 function DeploymentPage() {
-    const { deploymentId } = useParams();
+    const { deploymentId } = useParams() as { deploymentId: string };
```

Comment:
`useParams` returns `any` since we are on the old version of react-router. If `deploymentId` is not part of the URL, an empty string will be returned so this was the simplest method to get the types correct.